itest/lnd_route_blinding_test.go

```diff
@@ -1589,9 +1589,10 @@ func testPartiallySpecifiedBlindedPath(ht *lntest.HarnessTest) {
 		},
 	})
 
-	// Assert that it contains one path with 3 hops, with eve as the
-	// introduction node.
-	assertPathDetails(invoice, 3, eve.PubKey[:])
+	// Assert that it contains two paths are selected: one 3 hop one
+	// starting at Eve and a 3 hop one starting at Dave (this one will be
+	// padded with a dummy hop) in order to keep all the paths equidistant.
+	assertPathDetails(invoice, 3, eve.PubKey[:], dave.PubKey[:])
```

Comment:
I think we decided by design against this, why would you supply an incoming list, and then get a path which does not include those 3 channels. Open to change that but just want to point out that this is not a bug imo

Reply:
Ahh I was wrong thank you for clarification.